modules/data.remote/inst/s2ndvi.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Module to calculate NDVI from sentinel 2 data, based on Satellitetools by Olli Nevalainen.
+This will be used to create automated workflows for the remote data module.
+
+Warning: Point coordinates as input has currently not been implemented.
+
+Author: Ayush Prasad
+"""
+
+from satellitetools import gee
+import geopandas as gpd
+import os
+
+
+def s2_ndvi(geofile, outdir, start, end, qi_threshold):
+    """ 
+    Downloads sentinel 2 data, calculates NDVI and saves it in a netCDF file at the specified location.
+    
+    Parameters
+    ----------
+    
+    geofile (str) -- path to the file containing the coordinates of AOI, currently tested with geojson. 
+    
+    outdir (str) -- path to the directory where the output file is stored. If specified directory does not exists, it is created.
+  
+    start (str) -- starting date of the data request in the form YYYY-MM-DD
+    
+    end (str) -- ending date of the data request in the form YYYY-MM-DD
+    
+    qi_threshold (float) -- From satellitetools: Threshold value to filter images based on used qi filter. qi filter holds labels of classes whose percentages within the AOI is summed. If the sum is larger then the qi_threshold, data will not be retrieved for that date/image. The default is 1, meaning all data is retrieved
+        
+ 
+    Returns
+    -------
+    Nothing:
+            output netCDF is saved in the specified directory.
+            
+    Python dependencies required: earthengine-api, geopandas, pandas, netCDF4, xarray
+    
+    To test this function please run the following code, testfile is included.
+    
+    s2_ndvi(geofile="./test.geojson", outdir="./out/", start="2019-01-01", end="2019-12-31", qi_threshold=1)       
```

Comment:
could you move `test.geojson` into `satellitetools` folder and update this, not a big deal but that feels like a more appropriate place

Reply:
Sure.